src/editor.jsx

```diff
@@ -369,7 +409,7 @@ var Editor = React.createClass({
                     var duplicate = id in widgets;
 
                     widgets[id] = this.getWidgetEditor(id, type);
-                    var classes = (duplicate || !widgets[id] ? "error " : "") +
+                    classes = (duplicate || !widgets[id] ? "error " : "") +
                             (selected ? "selected " : "");
```

Comment:
Hmmm... thanks for moving the declaration to where it's actually declared! I think the fact that we do that indicates that there are larger problems with this function though--I don't like large multiple branches setting variables--it's hard to follow.